patches.json

```diff
@@ -180,6 +180,18 @@
     "Re-work consumers to terminate as soon as there is nothing left to process": {
       "2.2.0 - 2.3.1": "MAGECLOUD-4071__terminate_consumers_if_the_queue_is_empty__2.2.0.patch",
       "2.3.2 - 2.3.3": "MAGECLOUD-4071__terminate_consumers_if_the_queue_is_empty__2.3.2.patch"
+    },
+    "Reduce q-ty of error report files": {
+      "2.1.4 - 2.1.7": "MAGECLOUD-3392__reduce_q-ty_of_error_report_files__2.1.4.patch",
+      "2.1.8 - 2.1.16": "MAGECLOUD-3392__reduce_q-ty_of_error_report_files__2.1.8.patch",
+      "2.1.17 - 2.1.18": "MAGECLOUD-3392__reduce_q-ty_of_error_report_files__2.1.17.patch",
```

Comment:
will Magento have this fix in 2.1.19?

Reply:
I changed patches constraints to ~2.1.17 and ~2.2.8 as backport wasn't merged into 2.2.

Reply:
is it planned to be merged? if yes, then we will need to change this right after that Magento release

Reply:
@NadiyaS not until after the configuration flow is fixed. Just adding patches doesn't solve the issue itself - so there is additional work required on this task.